rqt_joint_trajectory_controller/rqt_joint_trajectory_controller/joint_trajectory_controller.py

```diff
@@ -170,19 +174,30 @@ def __init__(self, context):
         self._update_jtc_list_timer.timeout.connect(self._update_jtc_list)
         self._update_jtc_list_timer.start()
 
-        # subscriptions
-        subscribe_to_robot_description(self._node)
+        # Timer for running controller updates
+        self._update_robot_description_list_timer = QTimer(self)
+        self._update_robot_description_list_timer.setInterval(
+            int(1000.0 / self._ctrlrs_update_freq)
+        )
```

Comment:
this may be a little too fast, no? same frequency as the controller :/ should be capped to something fairly low in my opinion... or simply set to something low